.gitlab-ci.yml

```diff
@@ -0,0 +1,215 @@
+# Copyright 2018 Red Hat, Inc.
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# Part of pacemaker project
+# SPDX-License-Identifier: FSFAP
+
+variables:
+  GIT_DEPTH: "16"
+
+stages:
+  - buildenv
+  - build
+  - test
+
+.fedora-rawhide: &fedora-rawhide-anchor
+  tags:
+    - docker
+  image: fedora:rawhide
+
+.buildenv: &buildenv-anchor
+  stage: buildenv
+  variables:
+    GIT_STRATEGY: none
+
+.compilation: &compilation-anchor
+  only:
+   - pushes
+  # only since GitLab 11.4
+  #  changes:
+  #    - "configure.ac"
+  #    - "**/*.[ch]"
+  #    - "**/*.xml"
+  #    - "**/*.xsl"
+  #    - "**/Makefile.am"
+  #    - ".gitlab-ci.yml"
+
+buildenv:fedora-rawhide:info:
+  # primarily to gather some info about the image itself/container settings
+  <<: *buildenv-anchor  #extends: .buildenv
+  <<: *fedora-rawhide-anchor  #extends: .fedora-rawhide
+  script:
+   - cat /proc/meminfo | grep Free
+   - df -H
+   - lsblk
+   - nproc
+   - ulimit -a
+   - dnf --version
```

Comment:
maybe also pkg-config --modversion for libqb and corosync libraries

Reply:
Not here and not necessary IMHO, as long as the numbers match the
RPM versioning.

They can be observed in a parallel job where the packages get
"downloaded", or whenever they are installed, which are all
jobs order after this phase, except for `maint` now.